pkg/kubectl/cmd/set/set_env_test.go

```diff
@@ -22,14 +22,15 @@ import (
 	"strings"
 	"testing"
 
+	"os"
+
```

Comment:
Nit: No `gofmt`?

Reply:
not clean, but remove this null line gofmt will rework.